.pre-commit-config.yaml

```diff
@@ -6,6 +6,9 @@ ci:
   autoupdate_branch: ""
   autoupdate_commit_msg: "[pre-commit.ci] pre-commit autoupdate"
   autoupdate_schedule: quarterly
+  skip:
+    # requires 'jq' and 'yq', which dont' come pre-installed in the pre-commit.ci image
+    - generate-projects-to-versions
```

Comment:
This was something I hadn't considered when implementing this in a shell script and then introducing a `pre-commit` hook... for `language: system` types of hooks, you have to manage your own dependencies (https://pre-commit.com/#system).

And on `pre-commit.ci`, we don't have any ability (as far as I can tell) to pre-install things before `pre-commit` runs.

For now, to restore all the API docs, let's just skip this in CI.

I can put up a follow-up switching this to a Python script... that'd be easier to maintain anyway 😁